src/Tests/InstrEngineTests/InstrEngineTests/InstrEngineTests.csproj

```diff
@@ -502,9 +502,14 @@
     </Content>
   </ItemGroup>
   <ItemGroup Condition="Exists('$(ILAsmLocation)')">
-    <Content Include="$(ILAsmLocation)">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
+    <EmbeddedResource Include="$(ILAsmLocation)">
+      <Link>EmbeddedResources\ilasm.exe</Link>
+    </EmbeddedResource>
   </ItemGroup>
   <Import Project="$(EnlistmentRoot)\build\Managed.targets" />
+  <ItemGroup>
+    <EmbeddedResource Update="E:\dd\dm\CLRInstrumentationEngine\packages\microsoft.netcore.ilasm\7.0.0-preview.4.22229.4\runtimes\native\ilasm.exe">
```

Comment:
Do we need to use some sort of relative path here? My guess is this is specific to your machine

Reply:
This was a variable. VS must have replaced it with a full path when I saved the project. This is a bug in VS, I think.